src/mongoose_metrics.erl

```diff
@@ -328,7 +328,12 @@ ensure_metric(Host, Metric, Type, ShortType) when is_list(Metric) ->
     PrefixedMetric = name_by_all_metrics_are_global(Host, Metric),
     case exometer:info(PrefixedMetric, type) of
         ShortType -> {ok, already_present};
-        undefined -> exometer:new(PrefixedMetric, Type)
+        undefined ->
```

Comment:
Yep, race condition between ensure and create.
Occurs under heavy load, when metric is not created.
One process creates the metric, all other processes crash.